source/containers.tex

```diff
@@ -6086,6 +6086,12 @@
   // \ref{vector.bool}, class \tcode{vector<bool>}
   template<class Allocator> class vector<bool, Allocator>;
 
+  template<class T>
+    inline constexpr bool @\exposid{is-vector-bool-reference}@ = @\seebelow@;   // \expos
```

Comment:
It seems that the consensus is to drop `inline` from `constexpr` variable templates (#4625). Given this is exposition-only, I think it's totally OK to drop `inline`.

Reply:
done